proto/interchain_security/ccv/v1/shared_consumer.proto

```diff
@@ -0,0 +1,156 @@
+syntax = "proto3";
+
+package interchain_security.ccv.v1;
+
+option go_package = "github.com/cosmos/interchain-security/v3/x/ccv/types";
+
+import "tendermint/abci/types.proto";
+import "ibc/lightclients/tendermint/v1/tendermint.proto";
+import "ibc/core/channel/v1/channel.proto";
+import "google/protobuf/duration.proto";
+import "gogoproto/gogo.proto";
+import "interchain_security/ccv/v1/wire.proto";
+import "google/protobuf/timestamp.proto";
+
+//
+// Note any type defined in this file is referenced/persisted in both the consumer and provider CCV modules,
```

Comment:
nit: reading this `referenced/persisted in both the consumer and provider CCV modules,` I would have expected the file to be called `shared.proto`. My understanding is the file is called `shared**_consumer**.proto` because it contains _only_ consumer-related types used by the consumer but also referenced by the provider. If that's the case, might make sense to clarify in the comment.

Reply:
In this scenario, the file contains only consumer-related types used by the consumer but also referenced by the provider. This happens to also be the same set of types that are more generally shared between consumer/provider.

I can add a comment with this sort of this thing, or maybe it's worth clarifying once it contains only consumer-related types used by the consumer but also referenced by the provider is completed

Reply:
👍 So, I guess if this file would radically change as part of https://github.com/cosmos/interchain-security/issues/1214, then we can skip any changes here.